### ISSUE TITLE
Add support for EPC-QR-Code

### ISSUE THREAD

**Issue by Kamuno** [SecUSo/privacy-friendly-qr-scanner#98]
https://de.wikipedia.org/wiki/EPC-QR-Code

**Comment by ztNFny** [comment]
What's the feature request here? At least the example on Wikipedia is already scanned ok.

**Comment by Kamuno** [comment]
It's not about scanning the code but supporting it. Currently only the content text of the QR code will be displayed. It should rather be displayed in an ordered manner, like when scanning a contact.

**Comment by AnanasPfirsichSaft** [comment]
I agree. Generating such a code/image would be helpful when you are standing in front of a banking terminal (and prepared them before). At least you need Receiver, IBAN, Amount and Reason. Instead of EPC I encountered another format (ISO charset). A Sparkasse terminal can read both :)

`bank://singlepaymentsepa?name=DER%20EMPF%C4NGER&iban=DEXXXXXX&amount=3.49&reason=EINKAUF%20BEI%20ALDI&currency=EUR`

**Comment by qgustavor** [comment]
I should add that this app sometimes mistakenly detects Pix codes as URLs, which cause issues:

![Screenshot_2025-08-01-21-17-36-587_com.secuso.privacyFriendlyCodeScanner.jpg](https://github.com/user-attachments/assets/8501336c-10c9-48d1-b360-5311d1c0f6fc)

![Screenshot_2025-08-01-21-17-49-663_com.secuso.privacyFriendlyCodeScanner.jpg](https://github.com/user-attachments/assets/310f57af-72c7-494b-94b8-05888fde0fd0)

This was a one time payment QR code I scanned a few minutes ago which was mistakenly detected as a URL.

**Comment by issteve** [comment]
Meanwhile EPC-QR-Code is also known as Giro-Code in Germany and used more often. Having the possibility to generate such codes would be very useful and privacy friendlier then having to rely on e.g. a web service or closed source software.
An example could be found e.g. here: https://github.com/quasistatic-setup/EPC-QR-Code-Offline-Generator
